routes/users.js

```diff
@@ -45,4 +44,69 @@ router.get("/asset/:userid", verifyToken, async function (req, res, next) {
   }
 });
 
+router.post("/order/:userid", verifyToken, async function (req, res, next) {
+  try {
+    const { userid } = req.params;
+    const user = await User.findById(userid).lean().exec();
+
+    const userTransactionHistory = user.transactionHistory;
+    delete req.body.authorization;
+    const newTransactionHistory = req.body;
+    userTransactionHistory.push(newTransactionHistory);
+
+    const userCash = user.asset.cash;
+    const updatedUserCash = userCash + req.body.total;
```

Comment:
밑에 코드들을 보니 req.body.~~ 로 쓰는 값들이 많은 거 같아요

이 코멘트 또한 구조분해할당을 하시면 좀 더 가독성이 좋아질 거 같다는 생각이 들어서 남깁니다. !

```suggestion
    const { total, currencyName, unitsTraded ~~ } = req.body;
    const updatedUserCash = userCash + total;
```